vpc/tool/container2/setup_container_linux.go

```diff
@@ -145,35 +145,73 @@ func addIPv4AddressAndRoute(ctx context.Context, nsHandle *netlink.Handle, link
 	}
 
 	gateway := cidr.Inc(ip.Mask(mask))
-	logger.G(ctx).WithField("gateway", gateway).Debug("Adding default route")
-	newRoute := netlink.Route{
-		Gw:        gateway.To4(),
-		Src:       ip,
-		LinkIndex: link.Attrs().Index,
-	}
-	if mtu != nil {
-		newRoute.MTU = *mtu
-	}
-	err = nsHandle.RouteAdd(&newRoute)
-	if err != nil {
-		logger.G(ctx).WithError(err).Error("Unable to add route to link")
-		return errors.Wrap(err, "Unable to add default route to link")
+	var defaultRouteFound bool
+	if len(routes) > 0 {
+		for _, route := range routes {
+			if route.Family != vpcapi.AssignIPResponseV3_Route_IPv4 {
```

Comment:
What else could this possibly be? V6?

Reply:
In theory, we could add v6 support, but I haven't added custom MTU routes to V6 yet